src/components/Login/Login.js

```diff
@@ -1,8 +1,100 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
 import './Login.scss';
 
 const Login = () => {
-  return <div>Login</div>;
+  //입력한 pw 볼 수 있는 state
+  const [showPw, setShowPw] = useState(false);
+  const [userInfo, setUserInfo] = useState({ userId: '', userPw: '' });
+
+  function getUserInfo(e) {
+    const { name, value } = e.target;
+    setUserInfo({ ...userInfo, [name]: value });
+  }
+  //이메일 밑에 p 태그
+  const emailChk =
+    (userInfo.userId.includes('@') && userInfo.userId.includes('.')) ||
+    !userInfo.userId;
+  console.log(emailChk);
+  //비밀번호 밑에 p 태그
+  const pwChk = userInfo.userPw.length > 5 || !userInfo.userPw;
+
+  //링크 넘어가게
+  const isUserVaidate =
+    userInfo.userId.includes('@') &&
+    userInfo.userId.includes('.') &&
+    userInfo.userPw.includes.length >= 6;
+  const navigate = useNavigate();
+  const goToMain = () => {
+    navigate('/');
+  };
+
+  return (
+    <div className="loginScreen">
```

Comment:
```suggestion
    <div className="login">
```
가장 부모 태그는 컴포넌트의 이름과 동일하게 해주세요!